docs/guides/publisher-client-side.md

```diff
@@ -153,11 +161,19 @@ Depending on the structure of your code, it might be convenient to combine the c
 
 ### Bid Using UID2 Tokens
 
-Based on the status and availability of a valid identity, the SDK sets up the background token auto-refresh, stores identity information in [local storage or a first-party cookie](../sdks/client-side-identity.md#uid2-storage-format), and uses it to initiate requests for targeted advertising.
+Based on the status and availability of a valid identity, the SDK does the following:
+
+1. Sets up the background token auto-refresh.
+1. Stores identity information in [local storage or a first-party cookie](../sdks/client-side-identity.md#uid2-storage-format).
+1. Uses the identity information to initiate requests for targeted advertising.
+
+(**GWH_LP03 question. Not sure about the relationship between the steps above and the table below. And the diagram 2-a which says "the publisher calls the SSP for ads using the UID2 token". Any way you could help me understand/clarify? They don't seem entirely in sync.**)
```

Comment:
It looks like the diagram needs to be updated. I think maybe once we've finished any other changes, we should re-do it.

Reply:
Sounds good. No change here for now then! Thx.